tests/testData/parsing/variable_named_version_draft_2.wdl

```diff
@@ -0,0 +1,3 @@
+workflow a {
+  String version = "a"
+}
```

Comment:
https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file

Reply:
Ah, I inadvertently left this setting behind when I switched from Ultimate to Community - thanks